### ISSUE TITLE
Can't join team with get5 is activate.

### ISSUE THREAD

**Issue by AserCSGO** [splewis/csgo-practice-mode#102]
Hello,

I currently have a problem with your practice plugin when it is off and get5 is activate we can not join team. We must desactivate the get5 and then wait for everyone to join the server. I have it all look and I have no problem with get5.

**Comment by splewis** [comment]
I've regularly used get5+practicemode on the same server.

Can you provide version numbers (through the ``sm plugins`` command)? I'm not sure how to help you otherwise. 

Generally setting ``get5_kick_when_no_match_loaded`` in your get5 cfg should be enough.

**Comment by AserCSGO** [comment]
Hi,

Thanks for your answer,
The version is "CS: GO PracticeMode" (1.3.3 + 289) by splewis
"Get5" (0.6.0) by splewis

And get5_kick_when_no_match_loaded is set to 0.
More information : We can join the server but we can't join a team when the practicemode is disabled and the get5 plugin is activated.

**Comment by splewis** [comment]
I haven't tried reproducing with these versions yet, but do you have any other plugins active on the server? 

Posting the full plugin list from running ``sm plugins`` in console will help.

**Comment by AserCSGO** [comment]
"Fun Votes" (1.9.0.6245) by AlliedModders LLC
 "Nextmap" (1.9.0.6245) by AlliedModders LLC
 "Basic Info Triggers" (1.9.0.6245) by AlliedModders LLC
 "Player Commands" (1.9.0.6245) by AlliedModders LLC
 "CS:GO PracticeMode" (1.3.3+289) by splewis
 "Get5" (0.6.0) by splewis
 "Fun Commands" (1.9.0.6245) by AlliedModders LLC
 "Basic Commands" (1.9.0.6245) by AlliedModders LLC
 "Client Preferences" (1.9.0.6245) by AlliedModders LLC
 "Basic Ban Commands" (1.9.0.6245) by AlliedModders LLC
"csutils" (1.3.3+289) by splewis/Deathknife
 "Admin File Reader" (1.9.0.6245) by AlliedModders LLC
 "Anti-Flood" (1.9.0.6245) by AlliedModders LLC
 "Reserved Slots" (1.9.0.6245) by AlliedModders LLC
 "Admin Help" (1.9.0.6245) by AlliedModders LLC
 "Admin Menu" (1.9.0.6245) by AlliedModders LLC
 "Basic Votes" (1.9.0.6245) by AlliedModders LLC
 "Bot Mimic" (2.1) by Jannik "Peace-Maker" Hartung
 "Sound Commands" (1.9.0.6245) by AlliedModders LLC
 "Basic Chat" (1.9.0.6245) by AlliedModders LLC
 "Basic Comm Control" (1.9.0.6245) by AlliedModders LLC

**Comment by splewis** [comment]
I attempted to reproduce at those practicemode+get5 versions and was unsuccessful. 

Are there any errors logs in ``addons/sourcemod/logs``? Without hints from error logs there's probably nothing I can do here.

**Comment by AserCSGO** [comment]
L 11/17/2018 - 16:34:48: SourceMod error session started
L 11/17/2018 - 16:34:48: Info (map "de_overpass") (file "errors_20181117.log")
L 11/17/2018 - 16:34:48: Error log file session closed.
L 11/17/2018 - 20:56:57: SourceMod error session started
L 11/17/2018 - 20:56:57: Info (map "de_mirage") (file "errors_20181117.log")
L 11/17/2018 - 20:56:57: [get5.smx] Setting player auths in the "players" section has no impact with get5_check_auths 0
L 11/17/2018 - 20:58:35: SourceMod error session started
L 11/17/2018 - 20:58:35: Info (map "de_mirage") (file "errors_20181117.log")
L 11/17/2018 - 20:58:35: [get5.smx] Setting player auths in the "players" section has no impact with get5_check_auths 0
L 11/17/2018 - 20:58:45: [get5.smx] Setting player auths in the "players" section has no impact with get5_check_auths 0

**Comment by Bork1337** [comment]
did you solve this problem? I have same issue with practise-mode plugin

**Comment by AserCSGO** [comment]
No, but i think isn't due to practicemode. It's a command on the server.cfg

**Comment by Bork1337** [comment]
yeah did you make it work with practicemode on your server? my server can't seem to run it at the moment

**Comment by AserCSGO** [comment]
No, i'm sorry bro
